presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -359,6 +362,49 @@ public static Slice splitPart(@SqlType(StandardTypes.VARCHAR) Slice string, @Sql
         return null;
     }
 
+    @Description("creates a map using delimiter1 and delimiter2")
+    @ScalarFunction
+    @SqlType("map<varchar,varchar>")
+    public static Slice splitToMap(@SqlType(StandardTypes.VARCHAR) Slice string, @SqlType(StandardTypes.VARCHAR) Slice delimiter1, @SqlType(StandardTypes.VARCHAR) Slice delimiter2)
```

Comment:
Call the arguments `entryDelimiter` and `keyValueDelimiter` instead of `delimiter1` and `delimiter2`.